### PR TITLE
Remove dividers

### PR DESCRIPTION
**Pull request by dominik-zeglen**
This PR removes dividers from dialogs and adds stories for info dialog and dialog with table. Also adds utility components.

Screenshots:
Zrzut ekranu 2021-10-5 o 11 58 43
Zrzut ekranu 2021-10-5 o 11 58 50
Zrzut ekranu 2021-10-5 o 11 58 57

### CODE REVIEW

**Review by Droniu — APPROVED** [review]

File: generateFixture.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+const data = Array(argv.rows)
+  .fill(0)
+  .map(() => ({
+    name: faker.name.findName(),
+    email: faker.internet.email(),
+    job: faker.name.jobType(),
+  }));
+
+console.log(JSON.stringify(data));
```
Comment (jwm0) [inline generateFixture.js:21]: What's this file used for? It's not exporting anything

File: src/tools/useElementScroll.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import { throttle } from "lodash";
+import { useCallback, useEffect, useState } from "react";
+
+export type Position = Record<"x" | "y", number>;
+
+function getPosition<T extends HTMLElement>(
+  anchor?: T | null
+): Position | undefined {
+  if (!!anchor) {
```
Comment (jwm0) [inline src/tools/useElementScroll.ts:9]: Unnecessary casting `anchor` to boolean inside conditionals throughout this entire file
```suggestion
  if (anchor) {
```

File: stories/dialogs.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      <Button color="primary" variant="contained">
+        OK
+      </Button>
+    </DialogActions>
+  </Dialog>
+);
+
+export const WithOverflowTable: React.FC = () => {
+  const { anchor, position, setAnchor } = useElementScroll();
+  // isScrolledToTop can be undefined which is falsy, but we don'w want to
```
Comment (jwm0) [inline stories/dialogs.stories.tsx:42]:
```suggestion
  // isScrolledToTop can be undefined which is falsy, but we don't want to
```

**Review by bmigirl — APPROVED** [review]

File: generateFixture.js (hunk 1 above)
Comment (dominik-zeglen) [reply generateFixture.js:21]: Internal util, I'd rather have something like this in codebase than rewrite it over and over again after adding new component

**Pull request merged**